app/views/providers/start_merits_assessments/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= page_template page_title: t('.h1-heading') do %>
+  <%= simple_format t('.text') %>
```

Comment:
Can you please use meaningful names for your translation links. It makes the code a lot more readable, and content easier to identify. Something like `:do_merits_qualify_case` here for example.

Reply:
I've updated both the state machine and the locales file.